host_test.go

```diff
@@ -107,9 +117,7 @@ func TestHostNameCollision(t *testing.T) {
 		scanner := bufio.NewScanner(r)
 
 		// Consume the initial buffer
-		scanner.Scan()
```

Comment:
-0.5 on this change, not worth it unless we're doing this for large N (like >5).